contracts/viewers/FLIRebalanceViewer.sol

```diff
@@ -32,8 +32,12 @@ import { PreciseUnitMath } from "../lib/PreciseUnitMath.sol";
  * @title FLIRebalanceViewer
  * @author Set Protocol
  *
- * ETHFLI Rebalance viewer that returns whether a Compound oracle update should be forced before a rebalance goes through, if no
+ * FLI Rebalance viewer that returns whether a Compound oracle update should be forced before a rebalance goes through, if no
  * oracle update the type of rebalance transaction will be returned adhering to the enum specified in FlexibleLeverageStrategyAdapter
+ *
+ * CHANGELOG: 4/28/2021
+ * - Enables calculating Uniswap price with any multihop route
+ * - Lever and delever exchange data is read directly from strategy adapter and if not empty, calculate price based on encoded route
```

Comment:
While we're wonder if it makes sense to add the ability to switch the `router` is pointing towards